utils/src/main/java/ai/elimu/analytics/utils/BundleKeys.kt

```diff
@@ -0,0 +1,36 @@
+package ai.elimu.analytics.utils
+
+object BundleKeys {
```

Comment:
@tuancoltech Maybe add a comment here, warning that it's important to not rename any of these values because that would break backward-compatibility between the Analytics app and apps running different versions of its `utils` library.